docs/reference/requestBodies/add_or_edit_a_product.yaml

```diff
@@ -12,8 +12,24 @@ properties:
     type: string
     description: A valid corresponding password.
     example: mypassword
+  brands:
+    type: string
+    description: The brand of the product.
+    example: Häagen-Dazs
+  labels:
+    type: string
+    description: Product specific labels.
+    example: Kosher
+  categories:
+    type: string
+    description: The category of the product.
+    example: Desserts
+  packaging:
+    type: string
+    description: Packaging type, format, material.
+    example: Frozen
```

Comment:
we should also cite that there is an API v3 which add much more structured data for this field and should be privileged. with a link to https://openfoodfacts.github.io/openfoodfacts-server/reference/api-v3/#operation/post-api-v3-product-barcode

Reply:
fixed !